File: XCase.Swagger.ProxyGenerator/RAML/Tokens/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace XCase.REST.ProxyGenerator.RAML.Tokens
{
    class Comment : Token
    {
        /// <summary>
        /// Gets the value of the comment
        /// </summary>
        public string Value { get; private set; }

        /// <summary>
        /// Gets a value indicating whether the comment appears other tokens on that line.
        /// </summary>
        public bool IsInline { get; private set; }

        /// <summary>
        /// Initializes a new instance of the <see cref="Comment"/> class.
        /// </summary>
        public Comment(string value, bool isInline)
            : this(value, isInline, Mark.Empty, Mark.Empty)
        {
        }

        /// <summary>
        /// Initializes a new instance of the <see cref="Comment"/> class.
        /// </summary>
        public Comment(string value, bool isInline, Mark start, Mark end)
            : base(start, end)
        {
            IsInline = isInline;
            Value = value;
        }
    }
}

File: platform/src/java/com/xcase/rest/generator/raml/tokens/Comment.java
package com.xcase.rest.generator.raml.tokens;

import com.xcase.rest.generator.*;
import com.google.gson.*;
import com.xcase.common.impl.simple.core.CommonHTTPManager;
import com.xcase.common.utils.ConverterUtils;
import com.xcase.rest.generator.raml.EventType;
import com.xcase.rest.generator.raml.IParsingEventVisitor;
import com.xcase.rest.generator.raml.Mark;
import java.lang.invoke.MethodHandles;
import java.io.*;
import java.net.*;
import java.nio.file.*;
import java.util.*;
import org.apache.http.HttpEntity;
import org.apache.http.HttpResponse;
import org.apache.http.NameValuePair;
import org.apache.http.message.BasicNameValuePair;
import org.apache.http.util.EntityUtils;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class Comment extends Token {
    /// <summary>
    /// Gets the value of the comment
    /// </summary>
    public String Value;

    /// <summary>
    /// Gets a value indicating whether the comment appears other tokens on that line.
    /// </summary>
    public boolean IsInline;

    /// <summary>
    /// Initializes a new instance of the <see cref="Comment"/> class.
    /// </summary>
    public Comment(String value, boolean isInline) {
        this(value, isInline, Mark.Empty, Mark.Empty);
    }

    /// <summary>
    /// Initializes a new instance of the <see cref="Comment"/> class.
    /// </summary>
    public Comment(String value, boolean isInline, Mark start, Mark end) {
        super(start, end);
        IsInline = isInline;
        Value = value;
    }
}
